resources/js/Components/InputText/index.tsx

```diff
@@ -41,8 +42,8 @@ const InputText: React.FC<InputTextProps> = memo(({
             onChange={onChange}
             onBlur={onBlur}
             autoComplete={autoComplete ? autoComplete : "on"}
-            value={value || ""}
-            type="text"
+            value={type==="number"?value??0:value??""}
```

Comment:
Why not value || ""?
We will usually receive a string

Reply:
Because it can be input of number, but ok I'll do it